test/e2e-tests.sh

```diff
@@ -32,6 +32,11 @@ set +o pipefail
 echo ">> Deploying Pipeline CRD"
 ko apply -f config/ || fail_test "Build pipeline installation failed"
 
+# Make sure thateveything is cleaned up in the current namespace.
```

Comment:
This code is repeated here and in  e2e-tests.sh. You can factor out the apply and delete commands in a helper function in e2e-common.sh.

Reply:
makes sense, will change that